src/comm/Connection.h

```diff
@@ -149,6 +149,9 @@ class Connection : public RefCountable
     /** Netfilter MARK values currently sent on this connection */
     nfmark_t nfmark;
 
+    /** Netfilter CONNMARK value of this connection */
+    nfmark_t nfConnmark = 0;
```

Comment:
Did this before, but @yadij asked to initialize the variable in the header file :)

Reply:
> we are asking for opposite things

@gozzy, your call. I am not going to fight over this.

> initialization reduces the diff

All other factors being equal, code consistency trumps diff size IMO.

> carries over implicitly to any operators the compiler generates

This part of your claim does not compute at all for me:

1. Operators are not affected by member initialization. They only work on existing objects.
1. If you meant "copy constructor" instead of "any operators", then any such generated constructor would not use member default initialization because it will initialize each member using direct initialization. For this particular class, the generated copy constructor is deprecated since C++11. Similar reasoning applies to generated move constructors.
1. If you meant "default constructor" instead of "any operators", then any such generated constructor would be buggy in this inconsistent initialization case. For this particular class, no default constructor is generated, but its generation in other classes would indicate a bug.

Reply:
My inclination is to leave it in C++11 style and [maybe] later change the older code to this kind of initialization.